ee/clickhouse/views/groups.py

```diff
@@ -105,6 +108,18 @@ class Meta:
         fields = ["group_type_index", "group_key", "group_properties", "created_at"]
 
 
+class FindGroupSerializer(GroupSerializer):
+    notebook = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Group
+        fields = [*GroupSerializer.Meta.fields, "notebook"]
+
+    def get_notebook(self, obj: Group) -> str | None:
+        relationship = obj.notebook_relationships.first()
+        return relationship.notebook.short_id if relationship else None
```

Comment:
Can we avoid the N+1 here by doing something like `prefetch_related("notebook_relationships__notebook")` when finding the groups?